linode/manager
# M3-5531: Handle IPv6 Range Deletion

## Description

Add support for deleting IPv6 Ranges

## How to test

While pointed at dev, add IPv6 range through the API with something like: `curld networking/ipv6/ranges POST '{"linode_id":<linode_id>,"route_target":<route_target>,"prefix_length":64}'` , delete created IPv6 range in CM, confirm success
